lib/specifier.d.ts

```diff
@@ -1,4 +1,4 @@
-import type { parsed, Range } from "./shared";
+import type { Pep440Version, Range } from "./shared";
```

Comment:
Maybe we should also rename `Range` to `Pep440Range` ?

@rarkins WDYT?

Reply:
Good idea

Reply:
Or `Pep440Constraint` ?